Source/WebCore/Modules/streams/ReadableStream.cpp

```diff
@@ -92,22 +92,22 @@ ExceptionOr<Vector<Ref<ReadableStream>>> ReadableStream::tee(bool shouldClone)
 
 JSC::JSValue JSReadableStream::cancel(JSC::JSGlobalObject& globalObject, JSC::CallFrame& callFrame)
 {
-    return wrapped().internalReadableStream().cancelForBindings(globalObject, callFrame.argument(0));
+    return protectedWrapped()->internalReadableStream().cancelForBindings(globalObject, callFrame.argument(0));
```

Comment:
Why are you calling `protectedWrapped()` here and below? This is not what the warning was about.